### ISSUE TITLE
DMX Models not displaying properly in the Preview when grouped. 

### ISSUE THREAD

**Issue by spaulgal** [smeighan/xLights#3751]
When effects are placed on the models individually the preview is fine, but when the same effect is placed on the group of models it does not display properly. See attached package. 
I tried both the (DMX) 3D moving head model and the Area Floodlight model. The latter is currently set in the sequence. 
The actual outdoor display works fine. 
Note: When the render style is set to Default nothing happens on the Preview. When I change it to Default Per Model the preview does work with a simple On effect. 

Thanks

[Goodman St. Display 2.zip](https://github.com/smeighan/xLights/files/10982858/Goodman.St.Display.2.zip)

NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

**Describe the bug**

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)

**Expected behavior**
A clear and concise description of what you expected to happen and why.

**Screenshots**

**Versions (please complete the following information):**
 - OS: [e.g. Windows 10, Mac 10.14.5, etc...]
 - xLights version [e.g. 2019.37]

**Additional context**

**Attachments**
- If associated with a crash please attach/link to a dropbox containing the crash log that was produced
- If it crashed but no crash log was produced please use Tools Menu/Package Logs and attach/link to a dropbox containing the resulting zip file
- If the error relates to rendering an effect or a crash while sequencing please use Tools Menu/Package Sequence and attach/link to a dropbox containing the resulting zip file

If you would prefer not to post your sequences publically please say so and be ready to PM them to one of the developers when asked.

**Comment by AzGilrock** [comment]
Please read issue  #2350

**Comment by spaulgal** [comment]
Thanks Gil I was looking for another instance of this issue but didn't look back far enough. 

From reading the other thread it seems I cannot get the Preview to work using DMX models. I tend to use the plasma and other effects across a DMX group. I was hoping to use the Area Floodlight model because it offers a pretty good representation of how the actual display looks. 
Unless there's a better way, I'll use a Single Line model and just play with the pixel size and other perimeters to get it to look close. 

![1](https://user-images.githubusercontent.com/71677115/225431972-af238b86-7bb8-4faa-bf9b-56029607de91.jpg)

**Comment by AzGilrock** [comment]
It will work if you follow what I told him to do and he confirmed it worked.

**Comment by spaulgal** [comment]
Are you talking about changing the Layout Mode/Render Style? If so, I was able to get that to work much like you had it working in the video. The issue is I don't want all the lights in the DMX Group to be in unison. 

If you’re referring to your 2nd suggestion: "You can always create additional models that map to only the RGB channels and group those and treat them like normal lights"
....I'm going to have to wrap my head around how to do that.

**Comment by AzGilrock** [comment]
Yes DMX model groups can only be setup to get the first pixel colors of whatever effect is dropped so they will be in unison.   Once you create a group render buffer the effect routines don't have a way to figure out which model maps to which channels in the buffer they receive.  That's why only the "Per Model" option works because I added a mechansim to find the channel offset for a single model inside the rendering. 

The other option I mentioned just means create a Single Line model with 1 node and map its start channel to the RGB channels of the DMX model.  Create one for every DMX model.  Then group those models and they will act like a group of pixels.  It's basically like creating a shadow group.

There might be other ways to tackle it.

**Comment by spaulgal** [comment]
Ok I've used shadow models before. 
Just to be clear-- It'll be the Single Line models that will work as a group of pixels and there's no way at this point to have the DMX models do that?
